js/request.js

```diff
@@ -150,7 +150,6 @@ exports.request = function(req, res, opts, handler) {
       var terminationMessage = 'Terminating server at http://[localhost]:' + opts.port + ' for ' + opts['module-path'] + ' / ' + opts.handler;
       res.end(terminationMessage + '\n');
       console.info(terminationMessage);
-      server.close();
```

Comment:
It always struck me as odd to shut down the server via a request to the server. I'd prefer removing the DELETE handler entirely, and redefining LambdaRunner::Runner#stop (probably to do something like "@proc.kill".